src/aap_eda/services/project/imports.py

```diff
@@ -234,7 +248,22 @@ def _try_load_rulebook(
             content=content,
         )
 
-    def _is_rulebook_file(self, data: Any) -> bool:
+    def _validate_rulebook_file(self, data: Any) -> None:
```

Comment:
should we use the schema for validation? https://github.com/ansible/ansible-rulebook/blob/main/ansible_rulebook/schema/ruleset_schema.json